File: Pub-Sub-Pulsar/vendas/src/main/java/com/example/pulsar/vendas/event/SaleEvent.java
package com.example.pulsar.vendas.event;

import java.math.BigDecimal;
import java.util.List;

public class SaleEvent {
    private BigDecimal value;
    private String idStore;
    private List<String> products;
    private String adrres;
    private String idCliente;
    private String idSale;

    public SaleEvent(BigDecimal value, String idStore, List<String> products, String adrres, String idCliente, String idSale) {
    }
}

File: SQS-SNS/vendas/src/main/kotlin/com/sqs/service/vendas/event/SaleEvent.kt
package com.sqs.service.vendas.event

import java.math.BigDecimal

data class SaleEvent(
    val value: BigDecimal,

    val idStore: String,

    val products: List<String>,

    val adress: String,

    val idClient: String,

    val idSale: String,
)
